vpr/src/route/static_heap.h

```diff
@@ -0,0 +1,28 @@
+#ifndef _STATIC_HEAP_H
+#define _STATIC_HEAP_H
+
+#include "heap_type.h"
+#include "device_grid.h"
+
+t_heap* alloc_heap_data();
```

Comment:
Comment this is an interface to a single statically allocated heap that can be called from anywhere without an object pointer. This is useful to be compatible with existing vpr routing code.

Reply:
Done